docs/tap_hold.md

```diff
@@ -315,6 +315,16 @@ bool get_tapping_force_hold(uint16_t keycode, keyrecord_t *record) {
 }
 ```
 
+For an option to interrupt force hold, add the following to `config.h`:
+
+```c
+#define TAPPING_RELEASE_HOLD_TERM TAPPING_TERM - 100
```

Comment:
```suggestion
#define TAPPING_RELEASE_HOLD_TERM 100
```

and should probably add a check to make sure the define is less than the tapping term

Reply:
What would be a good way to warn about this during compile-time?

Reply:
Alternatively, we could say that the user is supposed to define the term with an offset relative to the tapping term and then in the code we do `TAPPING_TERM - TAPPING_RELEASE_HOLD_TERM`. That way, it's impossible to go above the tapping term.

Reply:
That is also not foolproof if user assigns a `TAPPING_RELEASE_HOLD_TERM` value that is larger than `TAPPING_TERM`, resulting with a negative timer value. Perhaps compile-time `#pragma message` warning might work here?